libs/sdk-ui-charts/src/highcharts/chart/HighChartsRenderer.tsx

```diff
@@ -199,6 +201,17 @@ export default class HighChartsRenderer extends React.PureComponent<
         });
     }
 
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    public onChartSelection(event: any) {
```

Comment:
- typing for `any`?
- add return type as `void`, and you can remove `explicit-module-boundary-types`

Reply:
I don't know the correct type of this event. I haven't found it in the HighCharts documents, so I left it as `any` type.

Reply:
How about `explicit-module-boundary-types`? can we remove it?

Reply:
It has a warning message when we check `yarn validate`, so I must disable the Eslint for this line.